packages/@azure/servicebus/data-plane/test/renewLockSessions.spec.ts

```diff
@@ -0,0 +1,689 @@
+import chaiAsPromised from "chai-as-promised";
+import dotenv from "dotenv";
+dotenv.config();
+chai.use(chaiAsPromised);
+import {
+  Namespace,
+  QueueClient,
+  generateUuid,
+  TopicClient,
+  SubscriptionClient,
+  ServiceBusMessage,
+  MessagingError,
+  OnError
+} from "../lib";
+import { delay } from "rhea-promise";
+import { testSessionId, purge } from "./testUtils";
+
+// Template starts
+
+// describe("Premium", function(): void {
+//   const PREMIUM_SERVICEBUS_CONNECTION_STRING = check(
+//     process.env.PREMIUM_SERVICEBUS_CONNECTION_STRING,
+//     "PREMIUM_SERVICEBUS_CONNECTION_STRING"
+//   );
+//   const namespace = Namespace.createFromConnectionString(PREMIUM_SERVICEBUS_CONNECTION_STRING);
+
+//   const PREMIUM_QUEUE = check(process.env.PREMIUM_QUEUE, "PREMIUM_QUEUE");
+//   describe("Unpartitioned Queues", function(): void {
+//     const senderClient = namespace.createQueueClient(PREMIUM_QUEUE);
+//     const sessionClient = senderClient;
+
+//     // Copy paste tests applicable for Unpartitioned Queues in Premium pricing here from the Standard section
+//   });
+
+//   const PREMIUM_TOPIC = check(process.env.PREMIUM_TOPIC, "PREMIUM_TOPIC");
+//   const PREMIUM_SUBSCRIPTION = check(process.env.PREMIUM_SUBSCRIPTION, "PREMIUM_SUBSCRIPTION");
+//   describe("Unpartitioned Topic/Subscription", function(): void {
+//     const senderClient = namespace.createTopicClient(PREMIUM_TOPIC);
+//     const sessionClient = namespace.createSubscriptionClient(PREMIUM_TOPIC, PREMIUM_SUBSCRIPTION);
+
+//     // Copy paste tests applicable for Unpartitioned Topic/Subscription Premium pricing here from the Standard section
+//   });
+// });
+
+describe("Standard", function(): void {
+  const SERVICEBUS_CONNECTION_STRING = check(
+    process.env.SERVICEBUS_CONNECTION_STRING,
+    "SERVICEBUS_CONNECTION_STRING"
+  );
+  const namespace = Namespace.createFromConnectionString(SERVICEBUS_CONNECTION_STRING);
+
+  const STANDARD_QUEUE_SESSION =
+    process.env.QUEUE_NAME_NO_PARTITION_SESSION || "unpartitioned-queue";
+  describe("Unpartitioned Queues", function(): void {
+    const senderClient = namespace.createQueueClient(STANDARD_QUEUE_SESSION);
+    const receiverClient = senderClient;
+    describe("Tests - Lock Renewal - Peeklock Mode", function(): void {
+      beforeEach(async () => {
+        await beforeEachTest(receiverClient);
+      });
+
+      afterEach(async () => {
+        await namespace.close();
+      });
+
+      it(`renewLock() with Batch Receiver resets lock duration each time.`, async function(): Promise<
+        void
+      > {
+        await testBatchReceiverManualLockRenewalHappyCase(senderClient, receiverClient);
+      });
+
+      it(`Receive a msg using Batch Receiver, wait until its lock expires, completing it now results in error`, async function(): Promise<
+        void
+      > {
+        await testBatchReceiverManualLockRenewalErrorOnLockExpiry(senderClient, receiverClient);
+      }).timeout(450000);
```

Comment:
This should no longer need a high timeout i.e after 
https://github.com/Azure/azure-sdk-for-js/commit/d987d2d835d4e6613908722b6a1180039532d044
We can update `testBatchReceiverManualLockRenewalErrorOnLockExpiry` to pass a small value for `maxSessionAutoLockRenewDurationInSeconds` to `getSessionReceiver`